src/main/java/org/wso2/carbon/module/swiftiso20022/mt/models/blocks/UserHeaderBlock.java

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.mt.models.blocks;
+
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field103;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field106;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field108;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field111;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field113;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field115;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field119;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field121;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field165;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field423;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field424;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field433;
+import org.wso2.carbon.module.swiftiso20022.mt.models.fields.Field434;
+
+
```

Comment:
Lets remove extra line

Reply:
Removed in a82da3f30afc3995c983ec8d68a04acfc363b6c0